submodules/users/users.js

```diff
@@ -3839,36 +3846,63 @@ define(function(require) {
 		usersCreate: function(data, success, error) {
 			var self = this;
 
-			self.callApi({
-				resource: 'user.create',
-				data: {
-					accountId: self.accountId,
-					data: data.user
+			monster.waterfall([
+				function(callback) {
+					self.usersCreateUser({
+						data: {
+							data: data.user
+						},
+						success: function(_dataUser) {
+							var userId = _dataUser.id;
+							data.user.id = userId;
+							callback(null, _dataUser);
+						},
+						error: function(parsedError) {
+							callback(parsedError);
```

Comment:
```suggestion
							callback(true);
```
We don't do actually handle any errors here, so passing the error payload is not really useful.